sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/parquet/ParquetTableProvider.java

```diff
@@ -39,19 +36,18 @@
  *   favorite_numbers ARRAY<INTEGER>
  * )
  * TYPE 'parquet'
- * LOCATION '/home/admin/users.parquet'
+ * LOCATION '/home/admin/orders/'
```

Comment:
Thanks for updating this, I guess the original version won't work since we always add a `/*` to the end for reads. This should probably just padd through the location directly instead, so the user can specify a glob if they want. Another follow-on jira I suppose.

Reply:
Agree this should be up to the user to specify exactly but there is an issue, you need a 'directory'  for the write so maybe we should specify this via some property like [they discuss here for Hive](https://stackoverflow.com/questions/11269203/when-creating-an-external-table-in-hive-can-i-point-the-location-to-specific-fil)? or do you have other idea(s)?